tests/kernel/fifo/fifo_timeout/src/main.c

```diff
@@ -367,7 +367,8 @@ static void test_timeout_fifo_thread(void)
 				&timeout, NULL,
 				FIFO_THREAD_PRIO, K_INHERIT_PERMS, 0);
 
-	packet = k_fifo_get(&fifo_timeout[0], timeout + 10);
+	packet = k_fifo_get(&fifo_timeout[0], timeout
+			    + k_ticks_to_ms_ceil32(2));
```

Comment:
Is k_ticks_to_ms_ceil32() available in master?

Reply:
look at the PR body, it says this Pr depends on #19591

Reply:
My fault.